metricbeat/module/postgresql/database/_meta/fields.yml

```diff
@@ -36,12 +36,12 @@
         that a read was not necessary (this only includes hits in the PostgreSQL
         buffer cache, not the operating system's file system cache).
     - name: blocks.time.read.ms
-      type: long
+      type: double
       description: >
         Time spent reading data file blocks by backends in this database, in
         milliseconds.
     - name: blocks.time.write.ms
-      type: long
+      type: double
```

Comment:
Please, double check if this change is problematic in a deployment with both versions of Metricbeat, we had recently some bad experiences with type changes that looked safe.

The kind of test you could do is:
* Install Metricbeat 7.15 and create a visualization using this field (even if it is zero).
* Double-check that you have a `metricbeat-*` index pattern where this field appears.
* In the same deployment, update to Metricbeat with this fix, and check if there is any problem with the visualization or with the index pattern.

Reply:
cc @kaiyan-sheng

Reply:
Yeah please double check here 😂 Not sure if it's the same case but I accidentally made a breaking change for changing field type from scaled float to long.